### PR TITLE
Readme update - can someone review?

### PR DESCRIPTION
**Pull request by devhenlee**
Sorry there's a small change to the error handling code in `createOrdersController.js`. I don't think it needs to be in a separate pull request...

### CODE REVIEW

File: controllers/createOrdersController.js
Comment (devhenlee) [inline controllers/createOrdersController.js:1]: Previously, the error message was "Out of stock". 

This change just makes it consistent with the new error message.

File: README.md [hunk 1]
```diff
@@ -305,14 +307,84 @@
...
+
+  * **Body Data**
+ 
+  ```json
+  { 
+     "order": {
+        "order_number": "ORDER0001",
+        "email_address": "jeff@amazeon.com",
+        "shipping_address": {
+            "customer_name": "Jeff Bayzos",
```
Comment (devhenlee) [inline README.md:350]: Changed the email address and customer name here so they don't match the real Jeff.

Comment (JamesMcGraaw) [reply README.md:350]: sounds good!

File: controllers/createOrdersController.js
Comment (JamesMcGraaw) [reply controllers/createOrdersController.js:1]: Yep this did need changing good catch

File: README.md [hunk 2]
```diff
@@ -287,13 +288,14 @@
...
+     "SKU": "SKU",
         {
-          "name": "String",
-          "stock_level": "Number"
+             "name": "Blunt Pencils",
+             "stock_level": 3
         }
   }
   ```
+**NOTE: `stock_level` takes positive *and* negative numbers **
```
Comment (JamesMcGraaw) [inline README.md:298]: great comment!

File: README.md [hunk 3]
```diff
@@ -305,14 +307,84 @@
...
+  ```json
+  {"message": "Successfully placed order"}
+  ```
+
+* **Error Response:**
+
+    * **Code:** 400 BAD REQUEST <br />
+      **Content:** 
+      `{"message": "Invalid order number - Already exists in database", "data": []}`
+      `{"message": "Not enough stock of Blunt Pencils", "data": []}`
```
Comment (JamesMcGraaw) [inline README.md:385]: does this need to be more generic rather than specifically saying blunt pencils?

**Review by JamesMcGraaw — APPROVED** [review]

**Pull request merged**